crates/templates/src/manager.rs

```diff
@@ -68,6 +68,8 @@ pub enum SkippedReason {
     AlreadyExists,
     /// The template was skipped because its manifest was missing or invalid.
     InvalidManifest(String),
+    /// The template name does match the provided IDs
```

Comment:
Is there meant to be a 'not' here?  Otherwise I'm unsure what this is saying.

Reply:
Yep - that is a typo. will fix.